src/components/wizard-navigation-bar.component.ts

```diff
@@ -69,6 +70,16 @@ export class WizardNavigationBarComponent {
     return this.wizardState.wizardSteps.length;
   }
 
+  /**
+   * Checks, whether a [[WizardStep]] can be marked as `completion` in the navigation bar
+   *
+   * @param wizardStep The wizard step to be checked
+   * @returns True if the step is a completion step
+   */
+  public isCompletionStep(wizardStep: WizardStep) {
+    return wizardStep instanceof WizardCompletionStepComponent;
```

Comment:
I don't understand, why we need to add a css tag to all completion steps of the wizard.
I mean normally we want to detect if a wizard has been completed, not if a wizard step is a completion step, right?

The first can be checked by using `wizardState.completed`.

Reply:
It's a leftover from trials. It's useless, I propose to remove it, especially if it causes issues.